values/cert-manager/cert-manager.gotmpl

```diff
@@ -33,12 +33,16 @@ startupapicheck:
 installCRDs: true
 
 resources:
+  {{- with $cm | get "resources" nil }}
```

Comment:
I would like to ged rid of defaults in values/*.gotmpl files.

Could you please move the
```
  requests:
     cpu: 50m
     memory: 64Mi
   limits:
     cpu: 200m
     memory: 384M
```
to default.yaml file ?
Then you can simply do
```
resources: {{- toYaml $cm. resources | nindent 2 }}
```
without extra logic

Reply:
Please don't for now. We have to split up that file as it becomes too big

Reply:
I envision a `defaults` folder with `$app.yaml` files